fao-house-style.csl

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>FAO House Style</title>
+    <title-short>FAO</title-short>
+    <id>http://www.zotero.org/styles/fao-house-style</id>
+    <link rel="self" href="http://www.zotero.org/styles/fao-house-style"/>
+    <link href="http://www.fao.org/docrep/004/ac339e/AC339E01.htm#P207_14907" rel="documentation"/>
+    <author>
+      <name>Bin Liu</name>
+      <email>lieubean@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <category field="social_science"/>
+    <category field="generic-base"/>
+    <summary>This style is created to meet the citation and bibliographical requirements of FAO House Style.</summary>
+    <updated>2015-08-22T21:47:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="verb-short">ed. by</term>
+      <term name="editor" form="short">
+        <single>ed</single>
+        <multiple>eds</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter="," suffix=".">
+      <name and="symbol" et-al-min="4" initialize-with="."/>
+      <label form="short" strip-periods="false" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
```

Comment:
could you set the et al mainly in cs:bibliography and cs:citation (where they're already included, you're just overriding them here). This is very hard to follow. You can leave the one in editor above if it's correct that you want et al for 4+ editors but only for 19+ authors in the bibliography.